crates/l2/Makefile

```diff
@@ -154,8 +156,12 @@ init-l2-no-metrics: ## 🚀 Initializes an L2 Lambda ethrex Client
 	--metrics.port ${L2_PROMETHEUS_METRICS_PORT} \
 	--evm levm \
 	--datadir ${ethrex_L2_DEV_LIBMDBX} \
-	--bridge-address ${BRIDGE_ADDRESS} \
-	--on-chain-proposer-address ${ON_CHAIN_PROPOSER_ADDRESS}
+	--bridge-address ${DEFAULT_BRIDGE_ADDRESS} \
+	--on-chain-proposer-address ${DEFAULT_ON_CHAIN_PROPOSER_ADDRESS} \
```

Comment:
What about putting these under a common category like `l1.bridge-address` & `l1.on-chain-proposer-address`?

Reply:
Nvm, changed my mind. I'll add this.

Reply:
Done in [1e3e72c](https://github.com/lambdaclass/ethrex/pull/2744/commits/1e3e72c5de8c0ea01e83463f3796bba3134bfef4).